### ISSUE TITLE
Using this asset on Linux

### ISSUE THREAD

**Issue by BastiaanOlij** [BastiaanOlij/godot-openvr-asset#3]
Unfortunately I'm not able to test the asset on Linux myself but there are now several people happily using it on Linux. There are a few gotcha's though.

The main one as I understand it is that you have to start Godot from Steam like so:
`$HOME/.steam/steam/ubuntu12_32/steam-runtime/run.sh /path/to/godot3executable`

There is a whole discussion going on the development Godot OpenVR issue list here:
https://github.com/BastiaanOlij/godot_openvr/issues/17

**Comment by beniwtv** [comment]
Yep, either you have to use the command above to launch Godot, or you can add it to your Steam library with Steam's "Add non-steam game to your library"-Function. (Not sure how they call it officially, the support site seems to be down.

**Comment by beniwtv** [comment]
So wrote a little more info we can include in the documentation.

Linux notes
=========

On Linux, SteamVR does not make it's libraries available system-wide. This means that if you run Godot (or your finished and exported game) outside of Steam, it will be unable to find these, and start without OpenVR support.

The easiest way to mitigate this is to run Godot (or your game) from within Steam, for example, if you are using the Steam version of Godot, or if you have added it as a shortcut in your library via the **Games -> Add a Non-Steam Game to my Library** menu.

Another option is to run Godot (or your game) in the Steam run-time environment. The command for this is:

    $HOME/.steam/steam/ubuntu12_32/steam-runtime/run.sh /path/to/Godot3orGameExecutable

In your exported games, you'll ideally not only want people to be able to run the game from within Steam (if your distributing your game there), but also directly from the game (or (an)other game launcher/store service(s)). In your game you could thus ship a script (maybe called **run.sh**) to run the game that would take care of running it in the Steam run-time:

```bash
#!/bin/bash
# Run this game in the Steam run-time, if we're not already running from within Steam

# Figure out our own path
exedir="$(cd "$(dirname "$0")" && pwd)"
# Change this to your game's executable name
exename="myGameExecutable"

# Steam will have set a STEAM_RUNTIME variable pointing at the runtime
if [ -z "$STEAM_RUNTIME" ]; then
    exec "$HOME/.steam/steam/ubuntu12_32/steam-runtime/run.sh" "$exedir/$exename" "$@"
    exitcode=$?   
else
    "$exedir/$exename" "$@"
    exitcode=$?
fi

exit $exitcode
```
